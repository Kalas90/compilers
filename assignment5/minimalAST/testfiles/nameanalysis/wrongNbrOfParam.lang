int main() {
    int a = func(1, 2);
}

int func(int a, int b, int c) {
    a = 1;
}