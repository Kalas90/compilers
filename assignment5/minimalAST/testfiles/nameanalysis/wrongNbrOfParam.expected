Error at line 2: Wrong number of parameters.
